README.md

```diff
@@ -7,34 +7,46 @@ The Backed community gives out NFTs to recognize community members and contribut
 
 The [first NFT type](https://opensea.io/assets/0xb9a069f85d51cc5cf23e910d1a5fd69744e926f1/1) was given out to those who used the product on Rinkeby in fall 2021. 
 
-In February 2022, we decided on some broad categories of community NFTs going forward, and various levels for each category. The current types of NFTs are 
+In February 2022, we decided on some broad categories of community NFTs going forward, and various levels for each category. Those types of NFTs were: 
 - **Trusted Tester**, given to those who use the product and catch a bug or provide some helpful feedback.
 - **Pawn Shopper**, given to those who are active in discord or community calls.
 - **Design Community**, given to those who contribute to design conversations and contribute design assets.
 - **Code Contributor**, given to those who contribute code that is deployed to production.
 
-New types will be added in the future as there is need. It is expected that each contributor will at most get a unique token/level once, e.g. you cannot get multiple Level 1 Trusted Tester tokens.
+In June 2022, we migrated to a new Community NFT that tracks XP across three categories: Activity, Contributor, and Community, and awards them both automatically (via bots) and through a nomination process. This rubric outlines eligibility for XP rewards: 
 
-We currently have three levels for each type and are working on fully fleshed out definitions. 
+### Automatically Awarded XP
 
-#### Trusted Tester
-- Level 1 Trusted Tester token, given to community members who tried the product and gave some useful feedback that was used to make a product change or caught an important bug. 
-- Level 2 Trusted Tester token, given to community members who have continually used the product and offered helpful feedback which has resulted in several product changes. 
-- Level 3 ...
+#### Activity XP
+- As a lender, provide your first loan (+1XP for 1st, 2nd, 4th, 8th)
```

Comment:
Maybe we should top out somewhere? Like 1st, 5th, 10th, and then every 10th from then on?

Reply:
It wasn't clear, but I was thinking exponential: 8th, 16th, 32nd, 64th... 
But! I also like 1st, 5th, 10th, and every 10th. Happy to go with that.